src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -348,7 +348,9 @@ public static string GetName(Type eventSourceType)
         /// which it is embedded.  This parameter specifies what name will be used</param>
         /// <returns>The XML data string</returns>
         public static string? GenerateManifest(
+#if !ES_BUILD_STANDALONE
```

Comment:
Just for my education - what does ES stand for?

Reply:
My guess would be **E**vent**S**ource.

Reply:
(These #defines are specific to EventSource implementation and not used anywhere else.)